emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/write/AbstractDynamoDBRecordWriter.java

```diff
@@ -118,8 +118,13 @@ public void write(K key, V value) throws IOException {
     if (result != null) {
       if (result.getConsumedCapacity() != null) {
         for (ConsumedCapacity consumedCapacity : result.getConsumedCapacity()) {
-          double consumedUnits = consumedCapacity.getCapacityUnits();
+          double consumedUnits = consumedCapacity.getTable().getCapacityUnits();
           totalIOPSConsumed += consumedUnits;
+          if (null != consumedCapacity.getLocalSecondaryIndexes()) {
+            for (String name : consumedCapacity.getLocalSecondaryIndexes().keySet()) {
+              totalIOPSConsumed += consumedCapacity.getLocalSecondaryIndexes().get(name).getCapacityUnits();
```

Comment:
Maybe consider move this part before `totalIOPSConsumed += consumedUnits;` and make it to be `consumed +=` to align with change in DynamoDBClient.java

Reply:
done